.github/workflows/test.yml

```diff
@@ -1,39 +0,0 @@
-name: Run tests
-on:
-  push:
-    branches:
-      - develop
-  pull_request:
-    branches:
-      - develop
-  # Allows to run this workflow manually from the Actions tab
-  workflow_dispatch:
-jobs:
-  test:
-    runs-on: ubuntu-latest
-    steps:
-      - name: "Checkout branch ${{ github.head_ref }}"
-        uses: actions/checkout@v3
-      - name: "Setup node"
-        uses: actions/setup-node@v3
-        with:
-          node-version: 16
-      - name: "Install the rust compiler"
-        run: |
-          curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-          source "$HOME/.cargo/env"
-          rustup default nightly
-          rustup component add rust-src --toolchain nightly-x86_64-unknown-linux-gnu
-      - name: "Install wasm-pack"
-        run: |
-          curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh -s -- -y
-      - name: "Install dependencies"
-        run: |
-          npm install
-      - name: "Build Aladin Lite"
-        run: |
-          npm run build
-      - name: "Run some tests"
-        run: |
-          npm run test:build
```

Comment:
I would not remove that. It runs some rust core tests on the core. There is only a few but it is still valuable to have them running on some CI.